### PR TITLE
feat: display gamescreen

### LINKED ISSUE #28: Display Gamescreen
The player wants to see the game screen to see which cards have been played by his opponents.

image

## Definition of done

- [ ] gamescreen widgets are created
- [ ] (optinal) also include in the middle of the widget what suit is currently the trump

### PR DESCRIPTION
**Pull request by BenjaminHac-Edu**
## This PR

- Adds the four different game boards

Closes: #28

### CODE REVIEW

**Comment by FRALEXAlexander** [comment]
I have issues with parts of the fragment cut off.
image
In 4 5 6 player view.
Are the lines in the cards intendet?

File: app/src/main/res/layout/fragment_gamescreen_five_players.xml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+            android:scaleType="fitCenter"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            app:srcCompat="@drawable/star" />
+
+        <androidx.fragment.app.FragmentContainerView
+            android:id="@+id/player1FragmentContainerView"
+            android:name="at.aau.serg.fragments.CardsFragment"
```
Comment (TannerGabriel) [inline app/src/main/res/layout/fragment_gamescreen_five_players.xml:27]: I don't quite understand why we use the `CardsFragment`, which displays a list of cards if we only want to display a single card.

File: app/src/main/java/at/aau/serg/fragments/GameScreenFivePlayersFragment.kt [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        // Inflate the layout for this fragment
+        return inflater.inflate(R.layout.fragment_gamescreen_five_players, container, false)
+    }
+
+    companion object {
```
Comment (TannerGabriel) [inline app/src/main/java/at/aau/serg/fragments/GameScreenFivePlayersFragment.kt:26]: We can delete this in all fragments if we don't plan to use the `newInstance` method and passing parameters.

File: app/src/main/res/layout/fragment_gamescreen_five_players.xml (hunk 1 above)
Comment (BenjaminHac-Edu) [reply app/src/main/res/layout/fragment_gamescreen_five_players.xml]: have now changed it to ImageViews

**Comment by FRALEXAlexander** [comment]
The lines in the cards are gone now the cutoff is still an issue.
Is this intendet?

**Comment by BenjaminHac-Edu** [comment]
> The lines in the cards are gone now the cutoff is still an issue. Is this intendet? 

It is not intendet. The problem was the hardcoded height of the FrameLayout

File: app/src/main/java/at/aau/serg/fragments/GameScreenFivePlayersFragment.kt (hunk 2 above)
Comment (BenjaminHac-Edu) [reply app/src/main/java/at/aau/serg/fragments/GameScreenFivePlayersFragment.kt]: is deleted

**Comment by FRALEXAlexander** [comment]
The cutoff is gone. Now the only thing is that the trick prediction is directly on top of the game all the times. But I think this is a issue for a different task. Am i right?

**Comment by BenjaminHac-Edu** [comment]
> The cutoff is gone. Now the only thing is that the trick prediction is directly on top of the game all the times. But I think this is a issue for a different task. Am i right?

I agree

**Review by FRALEXAlexander — APPROVED** [review]
LGTM

**Pull request merged**